development.ini

```diff
@@ -30,8 +33,8 @@ cache.list.backend = dogpile.cache.memory
 cache.list.arguments.cache_size = 5000
 cache.list.expiration_time = 7000
 
-sqlalchemy.url = sqlite:///%(here)s/atramhasis.sqlite
-#sqlalchemy.url = postgresql://atramhasis:atramhasis@localhost:5432/atramhasis
+#sqlalchemy.url = sqlite:///%(here)s/atramhasis.sqlite
+sqlalchemy.url = postgresql://postgres:postgres@localhost:5432/atramhasis
```

Comment:
I would prefer keeping the default db to sqlite. Any special reason for changing it to postgres?

Reply:
Not at all, I used postgres to be able to test and see more easily what's going on in the DB. I should've put sqlite back like I did in `test.ini`